### PR TITLE
Refactored Monitoring guide

### PR DESCRIPTION
**Pull request by RaekwonIII**
Added simpler Grafana config files

### CODE REVIEW

File: monitoring/README.md [hunk 1]
```diff
@@ -6,17 +6,23 @@
 
 # SSV - Monitoring
 
-`/metrics` end-point is exposing metrics from ssv node to prometheus.
+This page will outline how to monitor an SSV Node using Grafana and Prometheus.
+### Pre-requisites
+Make sure your node is exposing a `/metrics` and `/health` endpoints. This is done via node configuration, as explained in the [Installation guide on the docs]([../installation.md#metrics-configuration-optional](https://docs.ssv.network/run-a-node/operator-node/installation#create-configuration-file)).
```
Comment (andrew-blox) [inline monitoring/README.md:11]: Broken link

File: monitoring/README.md [hunk 2]
```diff
@@ -6,17 +6,23 @@
...
+
+For Grafana, specifically, [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) is a viable solution, especially for beginners.
 
 See the configuration of a [local prometheus service](prometheus/prometheus.yaml).
 
 ### Health Check
 
-Health check route is available on `GET /health`. \
+Even if Prometheus is not configured, the `/health` end-point can simply be polled by a simple HTTP client as a health check. \
 In case the node is healthy it returns an HTTP Code `200` with empty response:
```
Comment (andrew-blox) [inline monitoring/README.md:26]: `with empty response` -> `with an empty response`

Comment (andrew-blox) [reply monitoring/README.md:26]: Below:
`with HTTP Code 500` ->  `with an HTTP Code of 500`

File: operator/duties/scheduler_test.go [hunk 3]
```diff
@@ -237,3 +237,46 @@ func TestScheduler_Run(t *testing.T) {
 	cancel()
 	require.NoError(t, s.Wait())
 }
+
+func TestScheduler_Regression_IndiciesChangeStuck(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
```
Comment (andrew-blox) [inline operator/duties/scheduler_test.go:244]: In general this PR should be related to "Refactored Monitoring guide", please next time open a separate PR for separate tasks.

**Review by andrew-blox — CHANGES_REQUESTED** [review]

File: operator/duties/scheduler_test.go (hunk 3 above)
Comment (andrew-blox) [reply operator/duties/scheduler_test.go:244]: Ah sorry missed your comment above, no prob :)

**Comment by RaekwonIII** [comment]
Acted on changes requested.
Only thing now are the two "rogue" changes are harmless (can anybody confirm)...

**Review by nkryuchkov — COMMENTED** [review]
Great improvements, thanks! Just a couple of questions. Also, could you please rebase your commits onto stage? 

```
git checkout monitoring-page-refactor
git fetch
git rebase stage
git push --force-with-lease
```

Please let me know if you need any help with that

File: monitoring/README.md [hunk 4]
```diff
@@ -6,64 +6,111 @@
...
+
+```bash
+docker run \
+    -p 9090:9090 \
+    -v /path/to/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus
+```
+
+
+> ⚠️ Note: If you are not running Prometheus as a Docker container, but as an executable, change the \`targets\` in the config file to reflect the correct networking connections. In the case where the SSV Node container is called `ssv_node` the targets should look like this:
```
[the remark this answers is not in the document]
Comment (nkryuchkov) [reply monitoring/README.md]: @RaekwonIII thanks

**Review by nkryuchkov — APPROVED** [review]

**Review by andrew-blox — APPROVED** [review]

**Comment by lior-blox** [comment]
@RaekwonIII @zktaiga 

Can we combine https://github.com/bloxapp/ssv/pull/1145
into this pr?

**Comment by RaekwonIII** [comment]
I discussed it this morning with @zktaiga , we convened that he'll rebase his on top of my branch and eventually supersede this, I think.

**Comment by zktaiga** [comment]
> @RaekwonIII @zktaiga
> 
> Can we combine #1145 into this pr?

See https://github.com/bloxapp/ssv/pull/1176#issuecomment-1788565093

**Comment by zktaiga** [comment]
Superseded by #1176

**Pull request closed without merging**